compiler/qsc_doc_gen/src/generate_docs.rs

```diff
@@ -260,8 +261,12 @@ fn get_metadata(ns: Rc<str>, item: &Item, display: &CodeDisplay) -> Option<Metad
         ItemKind::Namespace(_, _) => None,
     }?;
 
+    let summary = parse_doc_for_summary(&item.doc)
+        .replace("\r\n", " ")
+        .replace('\n', " ");
```

Comment:
Why is replacing newlines with spaces needed here?

Reply:
They had said that newlines in the summary were not compatible with their tooling and so needed to be removed.